bin/sl-run.js

```diff
@@ -39,6 +48,7 @@ switch(config.profile) {
     // warning messages. If an app is missing a name, profile may still fail
     // to start, so drop-through to start(). We must re-supply options.
     if (agent().config.key)
+      tracer(tracerOptions);
```

Comment:
syntactially invalid! and in wrong spot, tracer has nothing to  do with our hosted strongops, should move to just before agent().start()